### ISSUE TITLE
SVG options are not optional

### ISSUE THREAD

**Issue by programbo** [pshihn/rough#79]
Thanks for adding TypeScript typings!

I just created a circle and wanted to set the `stroke` but discovered that the `options` parameter requires almost all fields to be supplied. Is this expected behaviour?

This is the interface for SVG Options:
```
export interface Options {
  maxRandomnessOffset: number
  roughness: number
  bowing: number
  stroke: string
  strokeWidth: number
  curveTightness: number
  curveStepCount: number
  fill: string | null
  fillStyle: string
  fillWeight: number
  hachureAngle: number
  hachureGap: number
  simplification?: number
}
```

For some reason, only `simplification` is optional.

**Comment by pshihn** [comment]
Your'e right. My mistake. I made it so because internally all values are available using default values. 
I did not consider people using these options. Will fix it shortly. (On holiday this week)
Thanks for reporting.

**Comment by programbo** [comment]
Cheers, @pshihn! Want me to just make a quick PR for `Options`? I haven't delved too much into roughjs yet, so I don't know of any other related issues, but I'll be happy to add some question marks and submit it as a PR 😉

**Comment by pshihn** [comment]
Resolved with https://github.com/pshihn/rough/pull/83

**Comment by pshihn** [comment]
Available in v2.2.4
